ios/RCTMGL/RCTMGLStyle.m

```diff
@@ -1847,6 +1847,11 @@ - (void)setIntensityTransition:(MGLLight *)layer withReactStyleValue:(RCTMGLStyl
 
 
 
+- (BOOL)_shouldAddImage:(NSString *)str
+{
+    return str != nil && ![self _isTokenString:str] && [self _isValidURL:str];
```

Comment:
Same comment: Do we still need `!isTokenString` when we already have `isValidURI`?